routes/vote.js

```diff
@@ -42,4 +43,41 @@ router.post('/', function(req, res, next) {
 
 });
 
+/* delete vote. */
+router.delete('/', function(req, res, next) {
+    console.log("url" + conf.backendUrl+voteUrl);
+    console.log("body" + JSON.stringify(req.body));
+    console.log("req.isAuthenticated()" + req.isAuthenticated());
+
+    if (req.isAuthenticated()) {
+
+        var ideaId = req.body.ideaId;
+        var personId = req.user.id;
+
+         var options = {
+           uri: conf.backendUrl + searchVoteUrl + "?personId=" + personId + "&ideaId=" + ideaId,
+           method: 'GET'
+         };
+
+        console.log("uri: "+ conf.backendUrl + searchVoteUrl + "?personId=" + personId + "&ideaId=" + ideaId);
+        request(options, function (error, response, body) {
```

Comment:
Please add a tiny bit of documentation here... I understand what you are doing but it took me a while to figure out.. basically you are searching for the person & id combination and using href to delete the vote.